tests/accessor/accessor_requisite_entire_buffer.cpp

```diff
@@ -0,0 +1,70 @@
+/*******************************************************************************
+//
+//  SYCL 2020 Conformance Test Suite
+//
+//  Provide checks that ranged accessor still creates a requisite for the entire
+//  underlying buffer
+//
+*******************************************************************************/
+
+#include "../common/disabled_for_test_case.h"
+#include "../common/get_cts_object.h"
+#include "catch2/catch_test_macros.hpp"
+
+// FIXME: re-enable when sycl::accessor is implemented
+#if !SYCL_CTS_COMPILING_WITH_HIPSYCL && !SYCL_CTS_COMPILING_WITH_COMPUTECPP
+#include "accessor_common.h"
+#endif
+
+namespace accessor_requisite_entire_buffer {
+
+DISABLED_FOR_TEST_CASE(hipSYCL, ComputeCpp)
```

Comment:
Could clarify why the test is disabled for these implementations?

Reply:
Code that doesn't work with these implementations is included via common file, now this code is disabled in common header file for hipSYCL and function that was disabled for ComputeCpp is enabled. So the test is enabled for all implementations now.